resources/themes/pterodactyl/server/tasks/index.blade.php

```diff
@@ -65,7 +65,13 @@
                                         <span class="label label-default">@lang('strings.no')</span>
                                     @endif
                                 </td>
-                                <td class="middle">{{ Carbon::parse($task->last_run)->toDayDateTimeString() }}<br /><span class="text-muted small">({{ Carbon::parse($task->last_run)->diffForHumans() }})</span></td>
+                                <td class="middle">
+                                @if($task->last_run > null)
```

Comment:
`@if($task->last_run)` should work here, or if you want to be extra verbose, `@if(! is_null($task->last_run))`